symtabAPI/src/emitElfStatic-ppc64.C

```diff
@@ -349,13 +350,13 @@ bool emitElfStatic::archSpecificRelocation(Symtab* targetSymtab, Symtab* srcSymt
 
 	if (relocation_length == 64) {
 		char *td = (targetData + dest - (dest%8));
-		uint64_t target = relocation;
-        	memcpy(td, &target, 2*sizeof(Elf64_Word));
+		auto const target = static_cast<uint64_t>(relocation);
```

Comment:
Should we use Elf64_LWord here or uint32_t  on line 357 for consistency?

Reply:
Probably. For now, I just wanted to keep the types as they were. I did just notice that I forgot to fix a couple items. I also want to add some `static_assert`s to make sure the type sizes are consistent since `setBits` traffics in `unsigned int` instead of `Elf64_Word`.